charts/argo-workflows/values.yaml

```diff
@@ -344,6 +344,9 @@ controller:
     enabled: false
     # minAvailable: 1
     # maxUnavailable: 1
+    
+    # -- [unhealthyPodEvictionPolicy] is only valid for v1.31+
```

Comment:
v1.31 of what?
It's valid for earlier versions but it's behind a gate.

Reply:
Fixed the comment ✅